fiatconnect-api.md

```diff
@@ -2235,7 +2236,7 @@ The `selfieDocument` and `identificationDocument` fields should be base64 encode
 
 ### 9.3.2. Fiat Account Schemas
 
-All Fiat Account Schemas supported by FiatConnect MUST contain the `accountName`, `institutionName`, and `fiatAccountType` fields. `accountName` is a friendly, user-definable name for the account.
+All Fiat Account Schemas supported by FiatConnect (except `PIXAccount`) MUST contain the `accountName`, `institutionName`, and `fiatAccountType` fields. `accountName` is a friendly, user-definable name for the account.
```

Comment:
seems like `accountName` is still just as useful for PixAccount as any other kind of account. Did you think `institutionName` is less useful because it's only for Brazilian Central Bank? If so, you could consider just adding the condition that `institutionName` must equal "Brazilian Central Bank". I'd prefer that over a 1-off exemption for PIXAccount for these required fields.

Reply:
I think you are right. I had understood these fields for another purpose. I will fix that

Reply:
thanks